mdata/chunk/sizes.go

```diff
@@ -0,0 +1,60 @@
+package chunk
+
+import (
+	"sync"
+)
+
+type Size uint8
+
+var ChunkSizes = map[Size]uint32{
```

Comment:
instead of a map you can simply use a `[32]uint32` (an array, not a slice, since it's static)
So something like
```
var ChunkSizes = [32]uint32{
1,
5,
10,
...
}
```

BTW see https://blog.golang.org/go-slices-usage-and-internals for some interesting implementation details of slices and arrays